packages/bfx-containers/src/functions/config.selectors.js

```diff
@@ -1,4 +1,4 @@
-export const apiUrl = process.env.REACT_APP_UFX_PUBLIC_API_URL || 'api.bitfinex.com'
+export const apiUrl = process.env.REACT_APP_UFX_PUBLIC_API_URL || 'api-pub.bitfinex.com'
```

Comment:
keep it as 'api.bitfinex.com', I believe not all `apiUrl/routes` will be supported by 'api-pub.bitfinex.com'
we can later on work on this and divide it into two parts apiUrl and publicApiUrl and use publicApiUrl for tickers